app/views/items/edit.html.erb

```diff
@@ -15,7 +15,7 @@
     <%= text_field_tag :image, "#{@item.image}" %>
 
     <%= label_tag :inventory %>
-    <%= text_field_tag :inventory, "#{@item.inventory}" %>
+    <%= number_field_tag :inventory, "#{@item.inventory}" %>
```

Comment:
I think this kind of change should be reflected in the `merchant/items` view, which is where items should be editable from